recipes/xtensor/all/conanfile.py

```diff
@@ -30,34 +29,24 @@ class XtensorConan(ConanFile):
         "openmp": False,
     }
 
-    @property
-    def _min_cppstd(self):
-        return "14"
-
-    @property
-    def _compilers_minimum_version(self):
-        # https://github.com/xtensor-stack/xtensor/blob/master/README.md
-        return {
-            "Visual Studio": "14",
-            "msvc": "190",
-            "gcc": "4.9",
-            "clang": "4",
-        }
-
     def export_sources(self):
         export_conandata_patches(self)
 
     def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("xtl/0.7.5")
+        # https://github.com/xtensor-stack/xtensor?tab=readme-ov-file#dependencies
+        if Version(self.version) < "0.26.0":
+            self.requires("xtl/0.7.5")
+        else:
+            self.requires("xtl/0.8.0")
         self.requires("nlohmann_json/3.11.3")
         if self.options.xsimd:
             if Version(self.version) < "0.24.0":
                 self.requires("xsimd/7.5.0")
             else:
-                self.requires("xsimd/13.0.0")
+                self.requires("xsimd/13.2.0")
```

Comment:
The upstream says xsimd/11.x for xtensor 0.25 (supported version in CCI). Is there any compatibility limitation by using 13.x directly?

Reply:
Website said 11.x, but the conanfile already required 13.0 for it. I didn't want to risk any breaking and assumed that minor version bumps don't contain breaking changes.

I can insert an `elif Version(self.version) < "0.26.0"` branch that keeps it pinned to `13.0.0`.

Reply:
aab3bdf3b96b478176b610f5d63a474db05cef9d

Adds this additional check. I can drop it again, if you want.